causal_testing/testing/causal_test_suite.py

```diff
@@ -37,3 +43,45 @@ def add_test_object(
         """
         test_object = {"tests": causal_test_case_list, "estimators": estimators_classes, "estimate_type": estimate_type}
         self.data[base_test_case] = test_object
+
+    def execute_test_suite(self, data_collector: ObservationalDataCollector,
+                           causal_specification: CausalSpecification) -> list[CausalTestResult]:
+        """Execute a suite of causal tests and return the results in a list
```

Comment:
I'm a little confused with what this is actually doing? Surely this should just be a `for` loop now of `test.execute()`, or am I misunderstanding what this class was meant for. Also it looks to be returning a dict not a list, so either the docstring or the returned object should be updated.

Reply:
Sorry for this making it in, I'm honestly not sure why it was using the `_return_causal_test_results` method rather than `execute_test`.